wpilibc/src/main/native/cpp/apriltag/AprilTagFieldLayout.cpp

```diff
@@ -83,15 +83,15 @@ bool AprilTagFieldLayout::operator!=(const AprilTagFieldLayout& other) const {
 
 void frc::to_json(wpi::json& json, const AprilTagFieldLayout& layout) {
   json = wpi::json{{"field",
-                    {{"length", layout.m_fieldLength.value()},
+                    {{"height", layout.m_fieldLength.value()},
```

Comment:
This was inconsistent between java and C++. I made the json file with java, so I chose its parlance as the source of truth

Reply:
It should be length.